idoit-install

```diff
@@ -1350,7 +1355,7 @@ EOF
         "$a2_en_mod" setenvif || abort "Unable to enable Apache module setenvif"
         log "Let every user read the logs"
         chmod 755 /var/log/apache2 || abort "Unable to change permissions"
-        chmod 664 /var/log/apache2/* || abort "Unable to change permissions"
+        chmod 664 /var/log/apache2/ || abort "Unable to change permissions"
```

Comment:
`664` on a directory seems not right

Reply:
true i accidentially removed the asterisk :<